src/common/loggers/csv_logger.cc

```diff
@@ -16,23 +17,22 @@
 
 namespace CsvLog {
 
-CsvLogger::CsvLogger(size_t log_id, const std::string& radio_name) {
+CsvLogger::CsvLogger([[maybe_unused]] size_t log_id,
+                     [[maybe_unused]] const std::string& timestamp,
+                     [[maybe_unused]] const std::string& radio_name) {
 #if defined(ENABLE_CSV_LOG)
   if (log_id >= kAllLogs) {
     AGORA_LOG_ERROR("Invalid log id %zu in CsvLogger\n", log_id);
   } else {
-    const std::string filename =
-        "files/log/log-" + kCsvName.at(log_id) + "-" + radio_name + ".csv";
-    std::remove(filename.c_str());  // delete file if already exists
+    const std::string filename = "files/log/" + timestamp + "/log-" +
+                                 kCsvName.at(log_id) + "-" + radio_name +
+                                 ".csv";
```

Comment:
Is spdlog making the /timestamp/ directory?

Reply:
Yes, spdlog creates new directory for output file.